fstest/test_all/config.yaml

```diff
@@ -395,6 +395,10 @@ backends:
  - backend:  "cache"
    remote:   "TestCache:"
    fastlist: false
+   ignore:
```

Comment:
This will cause the tests to get run and ignored which is fine but wastes CPU.

Perhaps we should expand the syntax of `config.yaml` slightly and add an `IgnoreTests` option?

https://github.com/rclone/rclone/blob/498d9cfa8550421a0b49092e8d0ed3a7eebd1fa2/fstest/test_all/config.go#L37

Reply:
Good point. I do think that would be a handy option. Or, if it's a bigger change than you want to make right now, I could handle it with a `t.Skip` in the bisync test code.

Reply:
In 84cebb6872cd8786e86357407f9159ec4386ba85 I merged an `ignoretests` parameter :-)